docs/content/config/user-management.md

```diff
@@ -82,5 +82,19 @@ Postfix supports so-called address tags, in the form of plus (+) tags - i.e. `ad
     recipient_delimiter = -
     ```
 
+### Send-Only Aliases
+
+Sometimes, it may be useful to allow certain accounts to send as other accounts, even when [the `SPOOF_PROTECTION` environment variable is enabled][spoof-protection]. This may be used to allow services to send accounts as other addresses/users without needing to disable spoof protection entirely, and without affecting incoming mail (which traditional [aliases](#aliases) would do).
+
+To configure these aliases, add them to `docker-data/dms/config/postfix-regexp-send-only.cf` in the same format as [the other regexp aliases](#configuring-regexp-aliases). For example:
+
+```cf
+/^.*@example.com$/ admin@example.com
+/^.*$/ superadmin@example.com
+```
+
+In this example, `admin@example.com` would be able to send as any address at `example.com` and `superadmin@example.com` would be able to send as any address at any domain.
```

Comment:
`superadmin@example.com` will not be able to send as `user@example.com` AFAIK, this is a caveat with regex lookup tables that you need to be aware of. The 1st match from top-down on the sender address will then check to the right-side value(s) for permitted logins (_which are often email addresses in DMS, but they could just be user names_).

You did not escape `.` for `.com` in the regex, in this example it shouldn't be a real issue, since alternative characters aren't going to produce a valid sender address, but it's still a caveat to be careful of:

`jan.doe@example.com` would also grant `janedoe@example.com` sender address to the login account, which might be a security problem when using the feature for accounts that aren't admins.

Reply:
Good catch on both counts; I'll clarify that in the documentation and warn about the caveats.

It looks like the following would allow `superadmin@example.com` to get the desired effect:
```suggestion
```cf
/^.*@example.com$/ admin@example.com superadmin@example.com
/^.*$/ superadmin@example.com
```

In this example, `admin@example.com` would be able to send as any address at `example.com` and `superadmin@example.com` would be able to send as any address at any domain.

...some notes about needing to order/etc
```